sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -1327,6 +1327,33 @@ object Radar {
         groups: Array<String>?,
         limit: Int?,
         callback: RadarSearchPlacesCallback
+    ) {
```

Comment:
Should we mark these as deprecated, and then remove them in a future version?  Same on ios?

Reply:
Do you mean deprecate the versions that don't take a `chainParameter` argument? Both platforms have a version with `chainParameter`, and one without, because in Objective-C, you can't have default arguments (and I didn't want to break existing calls), and even though Kotlin _can_ have defaults, Java _can't_, so we need the duplicates.

Reply:
Yeah, that's what I mean - mark the versions that don't take a `chainParameter` argument as deprecated.  Since we have the new version with the `chainParameter`, and the old ones are only there for backwards compatibility.

What do you think?